P202-Training.Domain/MappingService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using P202.Training.Data.Entities;
+using P202.Training.Domain.Models;
+
+namespace P202.Training.Domain
+{
+    public class MappingService : IMappingService
```

Comment:
Is this class being used? Which are the classes that consume it? I'm not seeing any service having an `IMappingService` injected

Reply:
No, this class is not necessary. I gonna remove this class.